### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cs-harsh-sharma**
#### Descriptions:
PMDB Link: https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=22386

UTCs:
 - Connector installation verified.
 - Connector logo verified.
 - Actions and Playbook list verified.
 - Playbook tags verified.
 - All Playbook are in info mode verified.
 - All Playbook are in inactive mode verified.

### CODE REVIEW

File: armis/connector.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        try:
+            operation = operations.get(operation)
+            return operation(config, params)
+        except Exception as err:
+            logger.error('{}'.format(err))
+            raise ConnectorError('{}'.format(err))
+
+    def check_health(self, config):
+        return _check_health(config)
+
```
Comment (cs-omkar-kumbhar) [inline armis/connector.py:26]: 100 X 100.png and 242 X 102.png images can be removed

File: armis/info.json [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+          "editable": true,
+          "visible": true,
+          "value": ""
+        }
+      ]
+    },
+    {
+      "operation": "add_tags_to_device",
+      "title": "Add Tags to a Device",
+      "description": "Adds tags to a Device.",
```
Comment (cs-omkar-kumbhar) [inline armis/info.json]: please see if better description can be added

File: armis/info.json [hunk 3]
```diff
@@ -0,0 +1,567 @@
+{
+  "name": "armis",
+  "label": "Armis",
+  "description": "Armis connector provides protection from cyber threats created by the onslaught of unmanaged and IoT devices. Get alerts and devices, update status of alerts and tag and untag devices.",
```
Comment (cs-prerna-joshi) [inline armis/info.json]:
```suggestion
  "description": "Armis connector provides protection from cyber threats created by the onslaught of unmanaged and IoT devices. This connector facilitates operations to get alerts and devices list, update status of alerts, tag and untag devices etc.",
```

File: armis/info.json [hunk 4]
```diff
@@ -0,0 +1,567 @@
...
+  "category": "Asset Management",
+  "version": "1.0.0",
+  "publisher": "Community",
```
Comment (cs-prerna-joshi) [inline armis/info.json]:
```suggestion
  "publisher": "Fortinet",
```

File: armis/info.json [hunk 5]
```diff
@@ -0,0 +1,567 @@
...
+        {
+          "title": "Tags",
+          "name": "tags",
+          "type": "text",
+          "tooltip": "The comma separated list of tags you want to add to the specified device.",
+          "description": "The comma separated list of tags you want to add to the specified device.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "placeholder": "Comma separated list of tags",
```
Comment (cs-prerna-joshi) [inline armis/info.json]:
```suggestion
          "placeholder": "e.g. Tag1, Tag2",
```

File: armis/info.json [hunk 6]
```diff
@@ -0,0 +1,567 @@
...
+          "value": ""
+        }
+      ]
+    },
+    {
+      "operation": "get_devices",
+      "title": "Get Devices",
```
Comment (cs-prerna-joshi) [inline armis/info.json]:
```suggestion
      "title": "Get Devices List",
```

File: armis/info.json [hunk 7]
```diff
@@ -0,0 +1,567 @@
...
+        "editable": true,
+        "visible": true,
+        "value": true
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_alerts",
+      "title": "Get Alerts",
```
Comment (cs-prerna-joshi) [inline armis/info.json]:
```suggestion
      "title": "Get Alerts List",
```

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**